### PR TITLE
Adding Initial project with License boilerplate

### PR DESCRIPTION
**Pull request by keshiha96**
Creating the initial base project of wso2am-k8s-operator which simply deploys the WSO2 API Manager and exposes the service. This mainly contains the artifacts folder where the configuration YAML files are defined, hack folder with files that generate the clientset, informers and listers folders, main.go file that has the controller logic and finally the controller.go that performs the actual logic and functionalities. Also, this project folder is copyrighted to WSO2 Inc. 2019, and the license boilerplate is defined for relevant files.

### CODE REVIEW

File: artifacts/examples/crd.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+kind: CustomResourceDefinition
+metadata:
+  name: apimanagers.apim.wso2.com
+spec:
+  group: apim.wso2.com
+  version: v1alpha1
+  names:
+    kind: Apimanager
+    plural: apimanagers
+  scope: Namespaced
```
Comment (dinusha92) [inline artifacts/examples/crd.yaml]: At a new line here.

File: controller.go [hunk 2]
```diff
@@ -0,0 +1,524 @@
...
+							Image: "wso2/wso2am:2.6.0",
+						},
+					},
+				},
+			},
+		},
+	}
+}
+
+func newService(apimanager *apimv1alpha1.Apimanager) *corev1.Service {
```
Comment (dinusha92) [inline controller.go:480]: Shall we add a method comment here.

Comment (keshiha96) [reply controller.go:480]: sure

File: artifacts/examples/crd.yaml (hunk 1 above)
Comment (keshiha96) [reply artifacts/examples/crd.yaml]: done

**Pull request merged**